projects/kit/components/input-file/file/file.template.html

```diff
@@ -62,15 +62,15 @@
         ></div>
     </div>
     <button
-        *ngIf="allowDelete && (fileTexts$ | async) as texts"
+        *ngIf="(allowDelete && (fileTexts$ | async)) as texts"
         automation-id="tui-file__remove"
         tuiIconButton
         type="button"
         size="xs"
         icon="tuiIconCloseLarge"
         appearance="icon"
         class="remove"
-        [title]="texts.remove"
+        [title]="$any(texts).remove"
```

Comment:
Why is this needed?

Reply:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/12021443/156890061-d7b16f24-ceba-4336-84d2-64447253ea23.png">

Reply:
fixed